src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1508,5 +1508,10 @@ internal enum ErrorCode
         ERR_RefReturnReadonlyNotField2 = 8411,
         ERR_ExplicitReservedAttr = 8412,
         ERR_TypeReserved = 8413,
+
+        //PROTOTYPE(ReadonlyRefs): make err IDs contiguous before merging to master. 
```

Comment:
There is not intent to make it available. I just do not want to clash with Omar's changes in the same file. Once we are done, we will "compact" the error codes.

---
In reply to: [121530313](https://github.com/dotnet/roslyn/pull/20142#discussion_r121530313) [](ancestors = 121530313)